go_tests/bal_test_test/test_outputs/test_report.txt

```diff
@@ -0,0 +1,17 @@
+Compiling source
+                foo/winery:0.1.0
+        
+        Running Tests
+        
+                winery
+                        [pass] testRunMain
+        
+        
+                        1 passing
+                        0 failing
+                        0 skipped
+        
+        Generating Test Report
+                /tmp/run-test-resources3803555381/target/report/test_results.json
```

Comment:
```suggestion
                /tmp/run-test-resources3803555381/target/report/test_results.json
```

This line won't work for windows since the file separator there is "\". You can have two files with windows and unix output and assert against the relevant file based on the OS.

Look for other such places as well.

Reply:
You can refer the existing resources to get an idea, you'll find two copies of each output text files for both OS.